### PR TITLE
Fix image render in `q.md` entry

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

Replaced relative path image link with direct, raw content URL to the same image.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/html/concepts/tags/terms/q/q.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ The `<q>` tag requires an opening and closing tag wrapped around a line of text.
 </p>
 ```
 
-![Example of what the rendered text would look like](../../../../../../media/q.jpg)
+![Example of what the rendered text would look like](https://raw.githubusercontent.com/Codecademy/docs/9b981e784a4869f6e903f4be2b5c3bd0ab724424/media/q.jpg)
```
Comment (yangc95) [inline content/html/concepts/tags/terms/q/q.md:41]:
```suggestion
![Example of what the rendered text would look like](https://github.com/Codecademy/docs/blob/main/media/q.jpg)
```

Comment (Dusch4593) [reply content/html/concepts/tags/terms/q/q.md:41]: Isn't that pointing to the page with the image as opposed to the image itself?

Comment (Dusch4593) [reply content/html/concepts/tags/terms/q/q.md:41]: This is what I'm getting when I try that:

Screen Shot 2022-03-04 at 1 11 26 PM

Comment (yangc95) [reply content/html/concepts/tags/terms/q/q.md:41]: what about: https://raw.githubusercontent.com/Codecademy/docs/main/media/q.jpg

Comment (Dusch4593) [reply content/html/concepts/tags/terms/q/q.md:41]: That works! 😄

**Review by yangc95 — APPROVED** [review]

**Pull request merged**